### ISSUE TITLE
pixel normal

### ISSUE THREAD

**Issue by chen990627** [microsoft/Deep3DFaceReconstruction#212]
`pixel_attributes,alphas = rasterize_triangles.rasterize_triangles(
      vertices, vertex_attributes, triangles, clip_space_transforms,
      image_width, image_height, [-1] * vertex_attributes.shape[2])

  pixel_normals = tf.nn.l2_normalize(pixel_attributes[:, :, :, 0:3], axis=3)`

the code above is from mesh_renderer.py and I debug it with a image.
why does the pixel normal contain negative values? what do these negative values represent?
